docs/fieldnorm/control_of_documents.md

```diff
@@ -1,25 +1,65 @@
-- Software development, and hence, corresponding documentation is prone to frequent updates. With the QMS of the laboratory, there are often waiting steps for different members of the laboratory to verify or authorize documentation. Hence, there might be a delay in release of a new version as the documentation of the previous release is not authorized yet.
-- add more
+Bioinformaticians work in a fairly different environement than biologists. The workflows they create, and the documentation that describe them, are all stored in files on a computer, and sometimes worked on by several person at the same time. So to follow the changes on both workflow and documentation, a more fitting system than the classic Quality Management System (QMS) offered for the biologists must be used. As specified in the ISO-15189, "Software procedures and related documentation needs to be controlled". This can be in any form or type of medium, as long as we achieve a certain set of requirements. In this document, we offer guidelines on how to comply with the norms while having a bioinformatician friendly QMS.
 
-To allow for a git-like QMS to be used by bioinformaticians the following aspects need to be considered, and documented within the QMS how compliance is achieved. ISO-15189:2022, 8.3.1
+## ISO norm requirements
 
-1. documents are uniquely identified;
-2. documents are approved for adequacy before issue by authorized personnel who have the =expertise and competence to determine adequacy;
-3. documents are periodically reviewed and updated as necessary;
-4. relevant versions of applicable documents are available at points of use and, where necessary, their distribution is controlled;
-5. changes and the current revision status of documents are identified;
-6. documents are protected from unauthorized changes and any deletion or removal;
-7. documents are protected from unauthorized access;
-8. the unintended use of obsolete documents is prevented, and suitable identification is applied to them if they are retained for any purpose;
-9. at least one paper or electronic copy of each obsolete controlled document is retained for a specified time period or in accordance with applicable specified requirements.
+To allow for a "Bioinformatician friendly" QMS, the following aspects need to be considered and documented :
 
-Examples are:
+1. Documents are uniquely identified
+2. Documents are approved for adequacy before issue by authorized personnel who have the expertise and competence to determine adequacy
+3. Documents are periodically reviewed and updated as necessary
```

Comment:
Should we have guidelines on how often this (minimally) happens?

Reply:
I would leave this open. As long as laboratories can reason why they choose a specific time period, this should be fine.

Reply:
Maybe we can add that it needs to be updated/checked at least whenever a new version/release of software/code/pipelines is pushed to production. Or maybe even better that the documents are checked as part of a validation/verification and hence before it is used for production?...